tests/tftp-tx-handling-rrq/suricata.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
```

Comment:
you should be able to leave this out as the default config has everything enabled already AFAICS